src/applications/simple-forms/26-4555/pages/contactInformation1.js

```diff
@@ -1,17 +1,31 @@
-import * as address from 'platform/forms-system/src/js/definitions/address';
+import { intersection, pick } from 'lodash';
+import fullSchema from 'vets-json-schema/dist/26-4555-schema.json';
+import addressUiSchema from 'platform/forms-system/src/js/definitions/profileAddress';
+import { veteranFields } from '../definitions/constants';
 
-import fullSchema from '../26-4555-schema.json';
+const { required, properties } = fullSchema.properties[
+  veteranFields.parentObject
+];
+const pageFields = [veteranFields.address];
 
-const contactInformation1 = {
+export default {
   uiSchema: {
-    address: address.uiSchema('Mailing address'),
+    [veteranFields.parentObject]: {
+      [veteranFields.address]: addressUiSchema(
+        `${[veteranFields.parentObject]}.${[veteranFields.address]}`,
+        undefined,
+        () => true,
+      ),
+    },
   },
   schema: {
     type: 'object',
     properties: {
-      address: address.schema(fullSchema, true),
+      [veteranFields.parentObject]: {
```

Comment:
@tlei123 One more main issue that we'll need to tackle tomorrow is the data structure of all the `schema`s and `uiSchema`s. Here is the updated file I worked on to remove the custom `profileAddress` component and use the forms-system definition instead. Notice how here in the `schema` and above in the `uiSchema` I updated the fields to be nested under `[veteranFields.parentObject]`. We have to do this to match the structure of the JSON schema in `vets-json-schema`. We'll need to make this change in all the pages or else we'll have issues when we ultimately hook up the submission functionality and authenticate with the big JSON schema.